### PR TITLE
Add underline to the "contribute a translation" link

### LINKED ISSUE #731: Translation banner link doesn't look like a link
## Description

The link to translations page on the translation banner looks just like text.

## Reproduction

1. Open `http://tr.wordpress.org/openverse'. 
2. You should see a translation banner inviting the users to contribute to translations, but the link looks just like the text around it:

Screen Shot 2022-01-27 at 5 30 15 PM

## Expectation

The link on the translation banner should be clearly identifiable.

## Screenshots

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #731 by @obulat

## Description

This PR adds an underline to the "contribute a translation" link in the banner for locales where translations are not finished. Together with the recent addition of pink color to all links, this makes the call-to-action link standout.
This PR also renames the components to add the `V` prefix, syncs the code style of the components, and makes the padding the same as the header padding. 

## Testing Instructions

The simplest way to see the translation banner is to set `v-show="!shouldHideBanner"` to the opposite: `v-show="shouldHideBanner"` (without the `!` in the condition).

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VNotificationBanner.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div
+    class="wrapper px-4 md:px-7 py-2 md:flex md:items-center md:justify-between"
+    dir="ltr"
```
Comment (krysal) [inline src/components/VNotificationBanner.vue:4]: Does this `dir` need to be always "ltr"? Doesn't this change for the Arabic language for example?

**Review by krysal — APPROVED** [review]
The issue is solved with this. Thanks for standardizing the component names also!

I just wonder why I can´t see this change just visiting http://localhost:8443/nl-be/search/?q=vermillion for example? 🤔 Looks like another problem tho.

**Review by zackkrida — APPROVED** [review]
CleanShot 2022-02-08 at 22 38 56@2x

The link looks nice! I do want to confirm, like @krysal suggested, that we're confident the banner is displaying when it should.

File: src/components/VNotificationBanner.vue (hunk 1 above)
Comment (obulat) [reply src/components/VNotificationBanner.vue:4]: I don't remember why this was added 🙈

**Pull request merged**